### ISSUE TITLE
[Linux] No desktop icon

### ISSUE THREAD

**Issue by bootsa** [graasp/graasp-desktop#135]
Desktop icon is not included/shown in Desktop Environment.
In panel chooser:
![image](https://user-images.githubusercontent.com/22413/61300063-96250700-a7e1-11e9-8d96-d7b7081525ba.png)
Or in task switcher:
![image](https://user-images.githubusercontent.com/22413/61299893-41818c00-a7e1-11e9-8b1c-27c63a997b30.png)

Linux Mint Cinnamon 19.1 (Tessa)

**Comment by juancarlosfarah** [comment]
Thanks for the feedback @bootsa! In the Ubuntu environment used to test the app, I do see the icon. May I ask how you installed the app?

**Comment by bootsa** [comment]
Sorry for the slow response.

I was running it straight from source as I had issues running the Snap image (see #143).

The AppImage build works but is also missing the icons.

It should still produce an icon even when run from source, correct?

There seems to be a few potential reasons for it:
- might need to be set in BrowserWindow's Icon option - see (this old but seemingly relevant comment)[https://github.com/electron/electron-packager/issues/128#issuecomment-227752065]
- it might be my distro/desktop environment (apparently Linux Mint / Cinnamon uses custom / third party icons rather than the app default - [from what I came across here](https://blog.linuxmint.com/?p=3773#comment-150031))

I'm seeing the same issue with a clean built Electron app. Will investigate further when I have the chance - though I guess this is a lower priority issue.

Are you seeing the correct icon @nunonun?

**Comment by bootsa** [comment]
Interestingly when opening the about window as fixed by @pyphilia the icon shows up for that window but not the main window (in both the dock and the tab switcher):
![image](https://user-images.githubusercontent.com/22413/61806451-fcd89f00-ae37-11e9-834b-c0972be8d8d9.png)
![image](https://user-images.githubusercontent.com/22413/61806654-5f319f80-ae38-11e9-9a30-29bb8cb30186.png)
It looks like because [About Window sets the BrowserWindow's icon option](https://github.com/rhysd/electron-about-window/blob/8ce5e3cffc56c0df6fd21071b6fce80b9b7333db/src/index.ts#L126)...

**Comment by bootsa** [comment]
Yep, simple as including `icon: ICON_PATH` in BrowserWindow options.

The image quality is rather poor when shown in the task switcher:

![image](https://user-images.githubusercontent.com/22413/61809081-e84ad580-ae3c-11e9-9820-5230016c1b97.png)

(I think the VS Code icon beside the Graasp icon is 1024 x 1024)

**Comment by juancarlosfarah** [comment]
@bootsa, thanks for investigating. Normally running using `yarn start` does not show the icon. However, running a built image should show the right icon. The low resolution seems weird, as in Ubuntu it looks fine. Let's try and get feedback from @NunonuN. @porduna, you also were able to install the app on Linux. Do you have any feedback you could give us?

**Comment by porduna** [comment]
Using the AppImage version, same icon issue, as well as in GNOME no option to add icon to favorites, etc.; however I guess that for desktop integration, I should install appimaged or AppImageLauncher according to https://docs.appimage.org/user-guide/run-appimages.html 

![image](https://user-images.githubusercontent.com/1900501/61822226-ee758d80-ae0d-11e9-993e-02df6390f0d7.png)

**Comment by bootsa** [comment]
The weird thing is that in the About window the icon comes out perfectly and is a lot larger than the task switcher version. 
![image](https://user-images.githubusercontent.com/22413/61951624-d3438300-afb1-11e9-839a-743bab6222ec.png)

On a HiDpi laptop screen that's automatically double scaled but same happens with no scaling (probably not apparent here):
![image](https://user-images.githubusercontent.com/22413/61951890-9deb6500-afb2-11e9-9005-f19c6eae965e.png)

OS: Linux Mint Cinnamon 19.1 Tessa
